packages/console/src/components/CardTitle/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { AdminConsoleKey } from '@logto/phrases';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  title: AdminConsoleKey;
+  subtitle?: AdminConsoleKey;
+};
+
+const CardTitle = ({ title, subtitle }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
```

Comment:
Should avoid using `AdminConsole` here, as this is a basic component. Make it more generic.

Reply:
for now it's a component for admin console only. we can change it when needed